test/index.spec.js

```diff
@@ -345,7 +345,7 @@ describe('CID', () => {
       const hash = await multihashing(Buffer.from(`TEST${Date.now()}`), 'sha2-256')
       const cid = new CID(1, 'dag-pb', hash)
       expect(cid.buffer).to.equal(cid.buffer)
-      expect(Object.hasOwnProperty('buffer')).to.be.false()
+      expect(Object.prototype.hasOwnProperty.call(Object, 'buffer')).to.be.false()
```

Comment:
I'm not sure what this line is (or was) trying to test...

Currently I think it's testing that the `Object` constructor doesn't have a `buffer` property...